### PR TITLE
implemented mockup of RelatedProducts comparison module. Created comp…

### PR DESCRIPTION
**Pull request by ZoyaStudio**
…onents for ProductCard and Carousel. Added more styles to styles.js in /related-products

### CODE REVIEW

**Review by alexandercausey — CHANGES_REQUESTED** [review]
Looks really good Elizabeth! I left a few comments about assumptions with the data for you to review and one about .map method. The last thing is a nit to pick, which is that you've got two spellings of Carousel, 'CardCarousel' in the filename and 'CardCarousol' in the function name :)

File: Client/src/Components/related-products/CardCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/extensions */
+import PropTypes from "prop-types";
+import React from "react";
+import css from "./styles.css";
+import ProductCard from "./ProductCard.jsx";
+
+function CardCarousol({
+  productInfo, title, buttonOnClickEvent, onClickEvent,
+}) {
+  let count = 0;
```
[the remark this answers is not in the document]
Comment (ZoyaStudio) [reply Client/src/Components/related-products/CardCarousel.jsx:10]: Cool, I will keep that in mind. The count variable here is just a placeholder for the product id will be unique with live data but isn't unique for the mockup data.

File: Client/src/Components/related-products/CardCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        onClickEvent={onClickEvent}
+      />
+    );
+  });
+  return (
+    <div className={css.carousel}>
+      <h3 className={css.thinHeading3}>
+        {title}
+      </h3>
+      {/* <button type="button" className={css.chevronLeft}>
```
[the remark this answers is not in the document]
Comment (ZoyaStudio) [reply Client/src/Components/related-products/CardCarousel.jsx:27]: It is circular, I just havn't added the logic for it.

File: Client/src/Components/related-products/RelatedProductsComparison.jsx [hunk 3]
```diff
@@ -1,8 +1,86 @@
...
+      },
+    },
+
+  ],
+};
+const formattedProduct = {
+  id: sampleProductDetails.id,
+  name: sampleProductDetails.name,
+  category: sampleProductDetails.category,
+  price: productStyles.results[0].original_price,
```
[the remark this answers is not in the document]
Comment (ZoyaStudio) [reply Client/src/Components/related-products/RelatedProductsComparison.jsx:66]: Ok, I will add logic for that

File: dist/main.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/*
+ * ATTENTION: The "eval" devtool has been used (maybe by default in mode: "development").
+ * This devtool is neither made for production nor for readable output files.
+ * It uses "eval()" calls to create a separate source file in the browser devtools.
+ * If you are trying to read the output file, select a different devtool (https://webpack.js.org/configuration/devtool/)
+ * or disable the default devtool with "devtool: false".
+ * If you are looking for production-ready output files, see mode: "production" (https://webpack.js.org/configuration/mode/).
+ */
+/******/ (() => { // webpackBootstrap
```
[the remark this answers is not in the document]
Comment (ZoyaStudio) [reply dist/main.js:9]: I'm not actually sure, I thought it was generated by webpack but I will omit it as it doesn't appear to have a function.

File: Client/src/Components/related-products/RelatedProductsComparison.jsx [hunk 5]
```diff
@@ -1,8 +1,86 @@
...
+  imgUrl: productStyles.results[0].photos[0].thumbnail_url,
+  rating: 3.5,
+};
+const relatedProductsArray = [formattedProduct, formattedProduct, formattedProduct,
```
[the remark this answers is not in the document]
Comment (ZoyaStudio) [reply Client/src/Components/related-products/RelatedProductsComparison.jsx:70]: It is just to help me format the carousel so it correctly displays cards when there are too many to fit on the screen.

**Comment by ZoyaStudio** [comment]
Closed because I need to fix some things

File: Client/src/Components/related-products/CardCarousel.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexandercausey) [reply Client/src/Components/related-products/CardCarousel.jsx:10]: Ah I see, so you won't need the index in the final implementation anyway :)

**Pull request closed without merging**